ironfish/src/rpc/routes/multisig/createSigningPackage.test.ts

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { ParticipantSecret, SigningCommitments } from '@ironfish/rust-nodejs'
+import {
+  createNodeTest,
+  useAccountFixture,
+  useMinerBlockFixture,
+  useUnsignedTxFixture,
+} from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+
+describe('Route multisig/createSigningPackage', () => {
+  const routeTest = createRouteTest()
+  const nodeTest = createNodeTest()
+
+  it('should create signing package', async () => {
+    const seed = 420
+
```

Comment:
There is quite a bit happening in this test. I feel like this the maximum we can add to a unit test. Subsequent tests we are going to either need fixtures or helper functions for setup.

Reply:
agreed, I think a multisig account fixture could be useful for a lot of these kinds of tests